File: lesson6/python3/homework/homework6_3.py
#!/usr/bin/python3
#: lesson6/python3/homework/homework6_3.py


def amount_digits_number(numb_n):
    count = 0 
    res = 1
    while count < numb_n:
        if numb_n != 0:
            numb_n //= 10
            res += 1
        count += 1
    return res


numb = int(input("Введите число -> "))
print('number of digits a of number ->', amount_digits_number(numb))

File: lesson6/c_c++/homework/homework6_3.c
/*
 * lesson6/c_c++/homework/homework6_3.c
 * Напишите функцию, которая вычисляет количество цифр числа.
 */
#include <stdio.h>

int amountDigitsNumber(int numb_n) {
    int count = 1;
    while ((numb_n /= 10) > 0) {
        count++;
    }
    return count;
}

int main(int argc, char* argv[]) {
    int numb;
    printf("Введите число -> ");
    scanf("%d", &numb);
    printf("number of digits a of number -> %d\n", amountDigitsNumber(numb));
    return 0;
}
